dashboard/modules/reporter/reporter_head.py

```diff
@@ -68,9 +67,6 @@ def __init__(self, dashboard_head):
         )
         self._gcs_aio_client = dashboard_head.gcs_aio_client
         self._state_api = None
-        self.thread_pool_executor = ThreadPoolExecutor(
```

Comment:
why do we remove this TPE and use the asyncio default one?

Reply:
Because that's what it's there for and we don't need another thread competing for GIL

Reply:
[Note to myself] we already have several thread pools scattered in dash code. Maybe we can make a global TPE for all dash needs.

Reply:
Yes, that single TPE should just be default event loop's one